nuclear_simulator/data_gen/config_engine/composers/comprehensive_composer.py

```
@@ -0,0 +1,417 @@
+Realistic Maintenance Test Composer
+
+This module provides a simplified composer that creates nuclear plant configurations
+with realistic industry-standard maintenance thresholds and targeted initial conditions
+to trigger specific maintenance actions through natural degradation.
+"""
+
+import sys
+import yaml
+from pathlib import Path
+from datetime import datetime
+from typing import Dict, List, Optional, Any, Union
```

Ruff (0.11.9)

`typing.Union` imported but unused

Remove unused import: `typing.Union`

(F401)